examples/demo/infrastructure_provisioning_with_i3s.rb

```diff
@@ -0,0 +1,81 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# This example is used to provision an Infrastructure with OS on Synergy with Image Streamer.
+
+my_client = {
+  url: ENV['ONEVIEWSDK_URL'],
+  user: ENV['ONEVIEWSDK_USER'],
+  password: ENV['ONEVIEWSDK_PASSWORD'],
+  api_version: 800
+}
+
+my_server_hardware_type = 'SY 480 Gen9 2'
+my_enclosure_group = 'SYN03_EC'
+
+# To create server profile template with i3s settings.
+oneview_server_profile_template 'SP-101-IS' do
+  client my_client
+  enclosure_group my_enclosure_group
+  server_hardware_type my_server_hardware_type
+  data(
+    osDeploymentSettings: { osDeploymentPlanUri: '/rest/os-deployment-plans/056041be-a7b3-478f-8e78-a37fd80b6654' },
+    connectionSettings: { manageConnections: true, connections: [
```

Comment:
well we have helper [function](https://github.com/HewlettPackard/oneview-chef/blob/bcdcd436203bc403c807eb0bd54bc76084615bd0/libraries/resource_providers/api200/server_profile_provider.rb#L16) to set connections. You are hard coding the url's which is not required if you use this method. But we need to revisit this function since its following 200 API request json structure.

Reply:
@soodpr Corrected, Its working perfectly.